src/cephadm/cephadm

```diff
@@ -82,6 +83,9 @@ if sys.version_info >= (3, 0):
 else:
     from urllib2 import urlopen, HTTPError
 
+if sys.version_info > (3, 0):
+    unicode = str
+
```

Comment:
this is a horrilbe hack, but allows py2 and 3. And the fact that in py2 the ipaddress needs to be unicode. If this only needs to be py3 only we can loose this.

Reply:
for now, `cephadm` is py3 only.

Reply:
cool, then we should remove py2 from the tox file! It's getting late here and on a Friday, so I'm calling it a night :)